### ISSUE TITLE
Autocomplete `xref` suggests version coord even if `true`

### ISSUE THREAD

**Issue by bartkl** [asciidoctor/asciidoctor-intellij-plugin#1708]
## Description

Hi,

I'm somewhat confused about the autocomplete behavior when cross-referencing pages in other component versions.

#### Playbook
Content sources section in my `antora-playbook-local.yml`:

```yaml
  sources:
    - url: .
      branches: HEAD
      start_path: ./root_component
    - url: ../doc-writing-guide/
      branches: HEAD
```

##### Root component version
The decriptor, i.e. contents of `antora.yml`:

```yaml
name: ROOT
title: Documentation
version: ~
nav:
- modules/ROOT/nav.adoc
```

##### _Writing Guide_ component version
The decriptor, i.e. contents of `antora.yml`:

```yaml
name: writing-guide
title: Writing Guide
version: true
prerelease: "-draft"
start_page: index.adoc
nav:
- modules/ROOT/nav.adoc
```

#### The problem
I'm editing the `ROOT/pages/index.adoc` file in my `ROOT` component, where I'm trying to cross-reference a page in my `writing-guide` component:

![image](https://github.com/user-attachments/assets/c65b6e1f-3b36-45b1-8c53-808d32f097f5)

As you can see from my typing, I don't want to refer to an explicit version here. Regardless, the autocomplete forces me to use that.

Is this expected behavior? Am I perhaps misunderstanding something?

Thanks for your help and this great plugin!

**Comment by ahus1** [comment]
Thank you for providing this example. I think you found two bugs: 

* The version `version` should never appear in autocomplete as it is a version which will be replaced by the Git refname
* The autocomplete should contain entries without a version number for component with the highest version number

I'll fix this in the next upcoming release. I'll let you know once a preview is available.

**Comment by ahus1** [comment]
The issue you reported should now be fixed in pre-release 0.43.2 of the plugin. I hope it now works for you as you expect.

The pre-release of the plugin is available from [GitHub releases](https://intellij-asciidoc-plugin.ahus1.de/docs/contributors-guide/user/installing-a-preview-release.html) and the [IntelliJ AsciiDoc EAP repository](https://intellij-asciidoc-plugin.ahus1.de/docs/contributors-guide/user/add-eap-repository-to-ide.html).

**Comment by bartkl** [comment]
Yes, it works fine now. Thanks for the fix!
